homeassistant/components/nmap_tracker/config_flow.py

```diff
@@ -107,6 +118,14 @@ async def _async_build_schema_with_user_input(hass, user_input):
             vol.Optional(
                 CONF_OPTIONS, default=user_input.get(CONF_OPTIONS, DEFAULT_OPTIONS)
             ): str,
+            vol.Optional(
+                CONF_TRACK_NEW,
+                default=user_input.get(CONF_TRACK_NEW, DEFAULT_TRACK_NEW_DEVICES),
+            ): bool,
+            vol.Optional(
+                CONF_SCAN_INTERVAL,
+                default=user_input.get(CONF_SCAN_INTERVAL, TRACKER_SCAN_INTERVAL),
+            ): vol.All(vol.Coerce(int), vol.Range(min=10, max=3600)),
```

Comment:
I don't think we need to expose these, but merely use them for import.

Reply:
I exposed it because otherwise the users only method to control it is to edit .storage/core.config_entries, and I didn't want to tell them to do that.  I figure it's inevitable that someone is going to ask how to change it since they had the ability to do it via yaml before.

If you feel strongly we shouldn't expose it, I'm also happy to take it out

Reply:
They can't change it, as it is part of the config flow, you can't retrigger that. So they still need to go into the config entries. This is not solving that problem.

Reply:
I'll suppress them doing the config flow and only show them in options.

Reply:
Retested with the sample yaml provided in the issue, and creating a new one.  All good